[2025-02-13T21:44:36.952+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-13T21:44:36.971+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2024-08-30T09:17:00+00:00 [queued]>
[2025-02-13T21:44:36.983+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2024-08-30T09:17:00+00:00 [queued]>
[2025-02-13T21:44:36.984+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-13T21:44:37.055+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_and_store_crypto_data> on 2024-08-30 09:17:00+00:00
[2025-02-13T21:44:37.066+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=17167) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-13T21:44:37.068+0000] {standard_task_runner.py:72} INFO - Started process 17175 to run task
[2025-02-13T21:44:37.065+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'crypto_data_collection', 'fetch_and_store_crypto_data', 'scheduled__2024-08-30T09:17:00+00:00', '--job-id', '21523', '--raw', '--subdir', 'DAGS_FOLDER/crypto_dag.py', '--cfg-path', '/tmp/tmpe6845nvr']
[2025-02-13T21:44:37.068+0000] {standard_task_runner.py:105} INFO - Job 21523: Subtask fetch_and_store_crypto_data
[2025-02-13T21:44:37.126+0000] {task_command.py:467} INFO - Running <TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2024-08-30T09:17:00+00:00 [running]> on host 7e1fe653b817
[2025-02-13T21:44:37.260+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_data_collection' AIRFLOW_CTX_TASK_ID='fetch_and_store_crypto_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-30T09:17:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-30T09:17:00+00:00'
[2025-02-13T21:44:37.261+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-13T21:44:37.328+0000] {logging_mixin.py:190} INFO - Erreur lors de la récupération des données CoinGecko : 429 Client Error: Too Many Requests for url: https://api.coingecko.com/api/v3/simple/price?ids=bitcoin%2Cethereum&vs_currencies=usd
[2025-02-13T21:44:37.328+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-13T21:44:37.339+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-13T21:44:37.339+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=crypto_data_collection, task_id=fetch_and_store_crypto_data, run_id=scheduled__2024-08-30T09:17:00+00:00, execution_date=20240830T091700, start_date=20250213T214436, end_date=20250213T214437
[2025-02-13T21:44:37.443+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-13T21:44:37.454+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
